### PR TITLE
Evmos 4 Pool FE Integration

### PR DESCRIPTION
**Pull request by ug02fast**
[DAI, FRAX, USDC, USDT] Pool

### CODE REVIEW

**Review by penandlim — APPROVED** [review]

File: src/constants/index.ts [hunk 1]
```diff
@@ -970,6 +991,7 @@ export const TBTC_POOL_TOKENS = [TBTC_V2, ...BTC_POOL_V2_TOKENS]
 export const TBTC_UNDERLYING_POOL_TOKENS = [TBTC_V2, BTC_SWAP_V2_TOKEN]
 
 export const EVMOS_BTC_POOL_TOKENS = [WBTC, RENBTC]
+export const EVMOS_4_POOL_TOKENS = [DAI, FRAX, USDC, USDT]
```
Comment (penandlim) [inline src/constants/index.ts:994]: The order of the tokens should be DAI, USDC, USDT, FRAX

https://github.com/saddle-finance/saddle-contract/pull/583/files

**Review by penandlim — CHANGES_REQUESTED** [review]

**Review by penandlim — APPROVED** [review]

File: src/components/TokenInput.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ function TokenInput({
 
   return (
     <div {...rest}>
-      {max != null && (
+      {max && (
```
Comment (hammeiam) [inline src/components/TokenInput.tsx:74]: wat

Comment (ug02fast) [reply src/components/TokenInput.tsx:74]: it's safer to avoid coercion. is there a reason we need to coerce?

Comment (nkrulikovsky) [reply src/components/TokenInput.tsx:74]: I think this should be optional.

**Pull request merged**